parsec/backend/postgresql/user_queries/find.py

```diff
@@ -88,6 +102,23 @@ async def query_find(
     return results, len(all_results)
 
 
+@query()
+async def query_retrieve_active_human_by_email(
+    conn, organization_id: OrganizationID, email: str
+) -> Optional[HumanFindResultItem]:
+    result = await conn.fetchrow(
+        *_q_retrieve_active_human_by_email(
+            organization_id=organization_id, now=pendulum_now(), email=email
+        )
+    )
+    if result:
+        return HumanFindResultItem(
```

Comment:
Though I think it's a good idea to introduce new classes to represent complex values, I think here we could avoid doing this now that we have removed the is_revoked field (and hence we can simplify by returning only a `Optional[UserID]` which would be enough for our current needs)